Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -3867,7 +3867,7 @@ Element* RenderLayer::enclosingElement() const
 Vector<RenderLayer*> RenderLayer::topLayerRenderLayers(const RenderView& renderView)
 {
     Vector<RenderLayer*> layers;
-    auto topLayerElements = renderView.document().topLayerElements();
+    auto& topLayerElements = renderView.document().topLayerElements();
```

Comment:
Do we even need this local variable?
Why can't we just do this?
for (auto& element : renderView.document().topLayerElements())

Reply:
Sure, fixed.